### PR TITLE
refactor(client): replace utils/UrlSafeBase64 with @silverhand/essentials/UrlSafeBase64

### PR DESCRIPTION
**Pull request by IceHe**
## Summary

- Reuse the implementation of UrlSafeBase64 from @silverhand/essentials
    - Update package `@silverhand/essentials` version from v1.1.0 to v1.1.1
    - Replace utils/UrlSafeBase64 with @silverhand/essentials/UrlSafeBase64
    - Remove utils/UrlSafeBase64 and its unit tests

## Testing

- Pass all existing unit tests

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/playground/package.json [hunk 1]
```diff
@@ -8,15 +8,14 @@
     "dev:razzle": "razzle start",
     "start": "concurrently -c \"blue,cyan\" \"pnpm:dev:tsc\" -k \"pnpm:dev:razzle\"",
     "start:prod": "NODE_ENV=production node build/server.js",
-    "build": "tsc -b && razzle build",
+    "build": "tsc -b && razzle build --noninteractive",
```
Comment (wangsijie) [inline packages/playground/package.json:11]: maybe is not a good solution

**Review by wangsijie — CHANGES_REQUESTED** [review]

File: packages/playground/package.json (hunk 1 above)
Comment (IceHe) [reply packages/playground/package.json:11]: I have google it but haven't found better solution so far.

Comment (IceHe) [reply packages/playground/package.json]: removed.

**Review by wangsijie — APPROVED** [review]

File: packages/client/webpack.config.js [hunk 2]
```diff
@@ -27,5 +27,9 @@ module.exports = () => {
     },
     mode: 'production',
     plugins: WITH_REPORT ? [new BundleAnalyzerPlugin()] : [],
+    performance: {
+      maxEntrypointSize: 394_000,
+      maxAssetSize: 394_000,
```
Comment (IceHe) [inline packages/client/webpack.config.js:32]: the seem not working?
image

Comment (simeng-li) [reply packages/client/webpack.config.js:32]: playground package exceed its limit, not client

Comment (IceHe) [reply packages/client/webpack.config.js]: My bad.

**Pull request merged**